source/console/pasdoc_Tokyo.dproj

```diff
@@ -1,141 +0,0 @@
-﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

```

Comment:
I am using Delphi Rio (10.3).
I didn't want to change your part and I don't know if you have specific Tokyo code.
Therefor I just generated Rio Projects.
But I do not do anything there.
So I can revert the changes - as you have specific "Tokyo" project I was misguided by reviewing your code.

Reply:
Yes, please revert these changes, so that project files for "Tokyo" stay in the repository.

You can push additional commits to GIT, and then they will be added to this pull request. Thank you!